rustfmt.toml

```diff
@@ -0,0 +1,102 @@
+# title: Ix Standard Rustfmt Configuration
+# author: John C. Burnham <john@argument.xyz>
+
+# For additional information about Rustfmt options see:
+# https://rust-lang.github.io/rustfmt/?version=v1.8.0
+#
+# General
+#version              = "One"              # rustfmt 2.0 is not released yet
+edition              = "2024"             # Rust edition
+
+#unstable_features    = false              # don't use unstable rustfmt features
+
+newline_style        = "Unix"             # Never allow Windows' \n\r newlines
+
+# disable_all_formatting = false            # mostly useless setting
+
+# Indent
+hard_tabs            = false              # hard tabs break consistent layout
+tab_spaces           = 2                  # override default 4 space indent
+#indent_style         = "Block"
+
+# Imports
+
+#imports_layout       = "Vertical"         # each import on its own line
+#imports_granularity  = "Crate"            # combine imports by crate
+reorder_imports      = true               # reorder from source
+#group_imports        = "StdExternalCrate" # in three groups
+
+# Width
+max_width            = 80                 # Don't go over 80 columns
+use_small_heuristics = "Max"              # Inherit max width for most objects
```

Comment:
This is a great way to make use of available width and minimize overly-vertical constructs 👍